examples/api/main.c

```diff
@@ -166,7 +171,7 @@ int main()
     {
         ok = API_LoadBallots(
             0, 
-            NUM_RANDOM_BALLOTS, 
+            MAX_BALLOT_PAYLOAD, 
```

Comment:
one of the failure cases that this PR resolves is the ability to call the LoadBallots endpoint with any value up to MAX_BALLOT_PAYLOAD.

this value will instead become an outparam in the future that will indicate the number of ballots actually read